android/src/main/kotlin/com/drenther/upi_pay/UpiPayPlugin.kt

```diff
@@ -94,29 +102,25 @@ class UpiPayPlugin internal constructor(registrar: Registrar, channel: MethodCha
     val uri = uriBuilder.build()
     val intent = Intent(Intent.ACTION_VIEW, uri)
 
-    val packageManager = activity.packageManager
+    val packageManager = activity?.packageManager
 
     try {
-      val activities = packageManager.queryIntentActivities(intent, PackageManager.MATCH_DEFAULT_ONLY)
+      val activities = packageManager?.queryIntentActivities(intent, PackageManager.MATCH_DEFAULT_ONLY)
 
       // Convert the activities into a response that can be transferred over the channel.
-      val activityResponse = activities.map {
-        val packageName = it.activityInfo.packageName
-        val drawable = packageManager.getApplicationIcon(packageName)
-
-        val bitmap = getBitmapFromDrawable(drawable)
-        val icon = if (bitmap != null) {
-          encodeToBase64(bitmap)
-        } else {
-          null
-        }
```

Comment:
Why was the null check guard removed?

Reply:
I'm not sure about this change will take a look maybe it got reformatted in Android studio